src/main.rs

```diff
@@ -116,10 +118,14 @@ fn main() -> Result<()> {
     }
     env_logger::init();
 
+    if args.info && args.path.exists() {
+        return info(&args.path);
+    }
```

Comment:
Should there be an error if `--info` is specified and there is no path argument?

Reply:
path is always required so that is caught before this, and if you get past here, there's another error to display if it doesn't exist. So we are covered.

Reply:
OK